Makefile

```
@@ -1,6 +1,6 @@
 SHELL := /bin/bash
 export GO111MODULE ?= on
-export VERSION ?= 1.0.0-bc10
+export VERSION ?= 1.0.0-bc14
```

Looking at the tags and how `go mod` operates, it should have the `v` prefix.
```suggestion
export VERSION ?= v1.0.0-bc14
```